config/src/config/state_sync_config.rs

```diff
@@ -153,3 +156,19 @@ impl Default for DataStreamingServiceConfig {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+pub struct DiemDataClientConfig {
+    pub timeout: Duration, // Timeout used by the network client when sending requests
+    pub data_summary_poll_interval: Duration, // Interval between data summary polls
+}
+
+impl Default for DiemDataClientConfig {
+    fn default() -> Self {
+        Self {
+            timeout: Duration::from_millis(10_000),
+            data_summary_poll_interval: Duration::from_millis(1_000),
+        }
+    }
+}
```

Comment:
nit: you'll probably want a new blank line after this. I suspect the linter might fail it? 😄

Reply:
Checked and made sure to have blank line.